backend/Dockerfile

```diff
@@ -1,17 +1,24 @@
 FROM mhart/alpine-node:12
 
+USER root
+
+RUN apk add --no-cache runit
```

Comment:
I use runit as a proper supervisor instead of a hacky sh script I used before in order to be able to run several backends in a single Docker container.